src/keysignature.ts

```diff
@@ -102,16 +116,40 @@ export class KeySignature extends StaveModifier {
     };
   }
 
+  // Add this key signature to the start of a stave.
   addToStave(stave: Stave): this {
     this.paddingForced = true;
     stave.addModifier(this);
 
     return this;
   }
 
+  // Get the `BoundingBox`
+  getBoundingBox(): BoundingBox {
+    if (!this.formatted) this.format();
```

Comment:
These one liners are completely fine in my opinion, because they're perfectly clear.

However it hurts readability when we do a one line if statement when the conditional and the statement are long and convoluted. It results in extra indentation and makes it prone to future bugs.

Reply:
ok